libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -61,6 +61,24 @@ class Strings {
         assertPrints(withoutMargin2, "XYZ\nfoo\nbar")
     }
 
+    @Sample
+    fun none() {
+        val empty = ""
+        val text = "abcdEFGH"
+
+        assertTrue(empty.none())
```

Comment:
In these two samples, it would be better to get rid of temporary variables and write something like `assertTrue("".none())`: these variables don't provide additional context, but a reader has to read their declarations first, remember them and only after proceed to actual use. So, calling `none()` directly on strings should simplify samples a bit.